pkg/declarativeconfig/configuration_test.go

```diff
@@ -0,0 +1,138 @@
+package declarativeconfig
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConfigurationFromRawBytes(t *testing.T) {
```

Comment:
Let's also make sure we have test that every configuration type gets decoded into correct proto type